packages/api/src/external/hie/set-doc-query-progress.ts

```diff
@@ -208,13 +208,14 @@ export function aggregateDocProgress(
   return tallyResults;
 }
 
-function aggregateStatus(docQueryProgress: DocumentQueryStatus[]): DocumentQueryStatus {
+export function aggregateStatus(docQueryProgress: DocumentQueryStatus[]): DocumentQueryStatus {
   const hasProcessing = docQueryProgress.some(status => status === "processing");
   const hasFailed = docQueryProgress.some(status => status === "failed");
+  const hasCompleted = docQueryProgress.some(status => status === "completed");
 
   if (hasProcessing) return "processing";
+  if (hasCompleted) return "completed";
```

Comment:
OOC - why would we want to lean towards completed? What if we had 4 HIE's and 3 failed and one completed.

Reply:
Because previously, if one failed it would spoil other perfectly good results. Maybe we can ticket to expose more of that to customers, but while debugging [why we were not sending success WHs to cxs](https://metriport.slack.com/archives/C04T256DQPQ/p1712356419967709).

Reply:
Gotcha i see - can just add this as a thought to the ticket we have to re-assess the doc query progress.